Formula/etcher-cli.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class EtcherCli < Formula
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher.git",
+    :tag => "v1.4.4",
+    :revision => "cbd531e161a12721fb763b526781f66ef8237ac9"
+
+  depends_on "python" => :build
+  depends_on "jq"
+  depends_on "node"
+
+  def install
+    Language::Node.setup_npm_environment
+    system "make", "RELEASE_TYPE=production", "cli-develop"
+    system "make", "RELEASE_TYPE=production", "package-cli"
+
+    inreplace "Makefile", "	mocha", "	npx -p node@6 mocha"
```

Comment:
You can clean this up by placing a space before and after mocha, you don't need all of them. See https://github.com/Homebrew/homebrew-core/blob/master/Formula/umlet.rb#L15 for an example.

Reply:
Its a tab not a space and the Makefile complains if I make it a space :(

Reply:
The fact that you have to run the tests with `node@6` should also indicate that if won't run against our `node` 10.